spec/index.html

```diff
@@ -812,15 +812,15 @@ <h3>Triple Terms</h3>
     </pre>
   </section>
 
-  <section id="reified-triples">
-    <h3>Reified Triples</h3>
+  <section id="reifying-triples">
+    <h3>Reifying Triples</h3>
 
-    <p>A <dfn data-cite="RDF12-CONCEPTS#dfn-reified-triple">reified triple</dfn>
+    <p>A <dfn data-cite="RDF12-CONCEPTS#dfn-reifying-triple">reifying triple</dfn>
       may be the <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a> or
       <a data-cite="RDF12-CONCEPTS#dfn-object">object</a> of an
       <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>.</p>
```

Comment:
I would consider this sentence as wrong now.

As per the definition in PR https://github.com/w3c/rdf-concepts/pull/98, a _reifying triple_ is an RDF triple whose predicate is `rdf:reifies` and whose object is a triple term. I don't think the intention here is to say that such an RDF triple may be the subject or object of an(other) RDF triple, but that's exactly what the sentence says now.

Reply:
The concepts are closely matched, and the `reifyingTriple` production can be used in the subject or object positions of another triple. Note that this is for Turtle triples, which result in triples in the abstract syntax which more closely align with the definition in RDF Concepts. Really `<< :s :p :o ~ :i >>` _is_ a reifying triple, as it results in `:i rdf:reifies <<( :s :p :o )>>` which is the Turtle/N-Triples expression closest to that defined in concepts.

Both `<< :s :p :o >> a rdf:Stated` and `:bob :said << :s :p :o >>` would be legitimate uses of that `reifiedTriple` production.

Reply:
I agree with Olaf's point. The subject or object of the other triple is the reifier id itself. It is not the `<< :s :p :o >>` item.

In that sense, the triple has been reified and then the reifier is used in the subject or object position.

Therefore I prefer `ReifiedTriple`.

Reply:
Very well, I change the term back to `reifiedTriple` but refer to "reifying triple" in RDF Concepts.

My point on subject/object was that, in the Turtle syntax, reifiedTriple can be used in either the subject or object position. Of course, this results in triples where it is the reifier that is in the subject or object position.

Reply:
> Really `<< :s :p :o ~ :i >>` _is_ a reifying triple

Rather, it is syntactic sugar for a reifying triple, that being `:i rdf:reifies <<( :s :p :o )>>`.

> Both `<< :s :p :o >> a rdf:Stated` and `:bob :said << :s :p :o >>` would be legitimate uses of that `reifiedTriple` production.

I think that those are uses of sugar for `reifyingTriple` productions, here for `[] rdf:reifies <<( :s :p :o )>> ; a rdf:Stated` and `:bob :said [] . [] rdf:reifies <<( :s :p :o )>>`. (The latter might be incorrect notation for the two instances of what must be the same blankNode, which had no identifier in the sugar statement, and so it seems should have no identifier in the de-sugared statement.)